modules/rest-api/pages/rest-set-up-services-existing-nodes.adoc

```diff
@@ -0,0 +1,140 @@
+= Assigning Services on an Existing Node
+
+:description: pass:q[Use the REST API to assign or remove services on an existing node in a cluster, and rebalance the cluster.]
+:page-topic-type: reference
+
+[abstract]
+{description}
+
+[#http-method-and-uri]
+
+=== HTTP Method and URI
+
+----
+POST /controller/rebalance
+----
+
+== Description
+
+Use the following REST API to add or remove non-data services on the existing nodes in a cluster.
+Then trigger a rebalance operation.
+
+=== curl Syntax
+
+You must specify all known nodes and the nodes to be ejected.
+
+----
+curl -v -X POST -u [admin]:[password] \
+http://[localhost]:8091/controller/rebalance \
+[-d knownNodes | -d ejectedNodes | -d topology]
+----
+
+These are services related parameters:
+
+* `knownNodes`: The value is a list containing all nodes that are in the cluster at the start of the rebalance process.
```

Comment:
`* `knownNodes`: The value is a list containing all nodes that are in the cluster.`